python/ray/train/__init__.py

```diff
@@ -2,14 +2,32 @@
 from ray.train.backend import BackendConfig
 from ray.train.data_config import DataConfig
 from ray.train.constants import TRAIN_DATASET_KEY
+from ray.train._internal.session import (
+    get_dataset_shard,
+    get_trial_resources,
+    get_world_rank,
+    get_world_size,
+    report,
+)
 from ray.train.trainer import TrainingIterator
 
+from ray.air import Checkpoint
+from ray.air.config import CheckpointConfig, RunConfig, ScalingConfig
 
 usage_lib.record_library_usage("train")
 
 __all__ = [
     "BackendConfig",
+    "Checkpoint",
+    "CheckpointConfig",
```

Comment:
Do we want to avoid moving the Checkpoint class if we likely will remove it?

Reply:
Alright, I will remove them.